src/Delegation.sol

```diff
@@ -94,6 +94,12 @@ contract Delegation is IDelegation, EIP712, GuardedExecutor {
         EnumerableSetLib.AddressSet approvedImplementations;
         /// @dev Mapping of approved implementations to their callers storage.
         mapping(address => LibStorage.Bump) approvedImplementationCallers;
+        /// @dev flags bitmap, which can be extended in the future to add boolean values.
+        LibBitmap.Bitmap flags;
+        /// @dev Address that can call the `pause` function.
+        address pauseAuthority;
+        /// @dev Last pause timestamp.
+        uint256 lastPauseTimestamp;
```

Comment:
Since these two fields will always be together, make `lastPauseTimestamp` into a uint40.

Reply:
I think calling it `lastPaused` feels more concise.